.gitignore

```diff
@@ -4,40 +4,48 @@
 .#*
 *.swp
 *.swo
-.DS_Store
-build
-#*
+*.zip
+*.nt.bz2
+*.tar.gz
+*.tgz
 *#
 
+.DS_Store
+
+.idea/
+.vscode/
+
 nilearn.egg-info/
-env/
+build
 dist/
-doc/.nojekyll
-doc/building_blocks/generated/
-nilearn_cache
-nilearn_data
-doc/_build/
-doc/auto_examples/
-doc/modules/generated/
-doc/datasets/generated/
-doc/generated/
+
 pip-log.txt
+
+env/
+venv/
+
 .coverage
 coverage
 nosetests_coverage/
-.ropeproject/
-tags
 
-*.zip
-*.nt.bz2
-*.tar.gz
-*.tgz
-
-.idea/
-venv/
+nilearn_cache
+nilearn_data
 
+doc/.nojekyll
+doc/_build/
+doc/auto_examples/
+doc/building_blocks/generated/
+doc/datasets/generated/
+doc/generated/
+doc/modules/generated/
 doc/themes/nilearn/static/jquery.js
-.vscode/
+
 examples/results/
 examples/**/results/
 examples/**/*.pdf
+
+.ropeproject/
+
+tags
+
+tmp
```

Comment:
All these changes seem unrelated . Shouldn't they go to another PR ?

Reply:
yup 
will put that on another branch

Reply:
See #3565